src/js/lib/converter.js.coffee

```diff
@@ -58,3 +59,13 @@ define "unit-converter", ["lodash"], (_) ->
 
     convert: (from, to, value) ->
       @CONSTANTS[from][to] * value
+
+  _.extend @OPL.Converter.Flowrate,
+    CONSTANTS:
+      "Standard Cubic Feet":
+        "Pounds": 0.0765
+        "Kilograms": 0.3048 ** 3 * 1.225
+        "Standard Cubic Meters": 0.3048 ** 3
+
```

Comment:
Refer to [here](https://en.wikipedia.org/wiki/Density_of_air)

Density of air:
*At sea level and at 15 °C air has a density of approximately 1.225 kg/m3 (**0.001225 g/cm3**, 0.0023769 slug/ft3, **0.0765 lb/ft3**) according to ISA (International Standard Atmosphere)*

Hence,
standard cubic feet (ft3) * 0.0765 (lb/ft3) = lb
standard cubic feet (ft3) * 0.3048 (m/ft) ** 3 * 1.225 (kg/m3) = kg
standard cubic feet (ft3) * 0.3048 (m/ft) ** 3  = m3